src/Deserialization/reader.jl

```diff
@@ -0,0 +1,7 @@
+struct TBReader
+    logdir::String
+end
+
+TBReadable = Union{TBLogger, TBReader}

```

Comment:
I’m not a fun of unions, as they are not extensible, but I think for now that is ok.,.